backend/Dockerfile

```diff
@@ -14,10 +14,10 @@ RUN cargo build --release
 ###
 ###
 
-FROM scratch
+FROM alpine:latest

```

Comment:
Why do you choose alpine here instead of scratch? As far as I know, lighter image is better for weight and security